mirrord/agent/src/file.rs

```diff
@@ -401,6 +408,18 @@ impl FileManager {
             })
     }
 
+    #[tracing::instrument(level = "trace", skip_all)]
+    pub(crate) fn read_link(
+        &mut self,
+        stripped_path: impl AsRef<Path>,
+    ) -> RemoteResult<ReadLinkFileResponse> {
+        let full_path = self.root_path.join(stripped_path);
+
+        read_link(full_path)
```

Comment:
Need to make sure the path we returned is normalized to be "from container" standpoint
Since we're basically calling readlink("/proc/PID/root/path..") I think it might return similar thing, which needs to be normalized..

Reply:
When I try this out:

```
ln -s "/gatos/rajado.txt" "/gatos/tigrado.txt"
readlink("/gatos/tigrado.txt"); // in C program
> "rajado.txt"
```

So we're not messing up the path, right?

Reply:
shouldn't it be `/gatos/rajado.txt`?

Reply:
It is, my example was off.

```
read_link:make_request_with_response: mirrord_layer::proxy_connection: return=Ok(Ok(ReadLinkFileResponse { path: "/gatos/rajado.txt" })) path=Success("/gatos/tigrado.txt") request=ReadLinkFileRequest { path: "/gatos/tigrado.txt" }
```